apps/pip/account-management-inactive-verification-cron/demo/00.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: pip-account-management-inactive-verify-cron
+spec:
+  releaseName: pip-account-management-inactive-verify-cron
+  values:
+    job:
+      schedule: "01 08 * * 0,2,4,6"
+      environment:
+        DATA_MANAGEMENT_URL: https://pip-data-management.demo.platform.hmcts.net
+        ACCOUNT_MANAGEMENT_URL: https://pip-account-management.demo.platform.hmcts.net
+        TRIGGER_TYPE: ACCOUNT_INACTIVE_VERIFICATION
+    global:
```

Comment:
You might not need these to my knowledge

Reply:
Thanks. I've removed these from 'test' (except for jobKind: CronJob) to try that out. If it works in that env once deployed I'll go through the other env's and remove them too :)